packages/core/src/actions/types.ts

```diff
@@ -147,6 +147,7 @@ export type TransactionEstimatedGas = {
 export type NetworkGasEstimate = {
   chainId: number
   estimatedGas: string
+  fundsRequested: string
```

Comment:
When we estimate the deployment cost, we need to know the amount of funds requested on each network. We already track that in the compiler config, but it is uploaded separately from the proposal endpoint, making it somewhat difficult to access. Since the compiler config can be quite a large file, downloading it from S3 in the proposal endpoint is not super practical. 

So I just included the amount of funds requested here in addition to in the compiler config which simplifies things a lot. This does introduce two sources of truth, though, which is unfortunate. I've updated the proposal test suite to include a check that these values are always identical.